docs/administrators/operational_procedures_with_lsm.rst

```diff
@@ -1,9 +1,9 @@
-.. only:: iso
+.. _operational_procedures_lsm:
 
-    .. _operational_procedures_lsm:
+Operational Procedures With LSM
+--------------------------------
 
-    Operational Procedures With LSM
-    --------------------------------
+.. only:: iso
```

Comment:
I don't think we need the `only` directive here and on the other pages that are iso only, because those pages are excluded using the `exclude_patterns` configuration option. Like that we would have two way to exclude things from oss:

* The `exclude_patterns` configuration option if a whole page needs to be excluded.
* The `only` directive if need to exclude only parts of a page.

Combining both mechanisms on a single page is not very useful I think. Wdyt?

Reply:
> Combining both mechanisms on a single page is not very useful I think. Wdyt?

I agree, my rationale was to have extra safety to make sure the iso-only content is not shown e.g. if we inadvertently change the exclude config of the conf.py file. I think it is more confusing than useful, I'll remove it